core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingAction.java

```diff
@@ -112,6 +112,9 @@ public class SimpleReportingAction implements IStreamProcessingAction, IStreamin
 
   public static final String RESERVEDMAPKEY_LINEAGE_ID = "lineage-id";
 
+  public static final String REPORT_EXTERNALIZE_STYLE = "externalize-style";
+  public static final String REPORT_BASE64_IMAGES = "base64-images";
```

Comment:
See comment related to these String constants https://github.com/pentaho/pentaho-scheduler-plugin-ee/pull/244/files#r1626298970

Reply:
I'm fine with the variables. I'm worried about introducing a cyclic build between pentaho-scheduler and the reporting projects.

The reporting projects - https://github.com/pentaho/pentaho-platform-plugin-reporting and https://github.com/pentaho/pentaho-reporting don't have an API jar.

I just discovered "useJcr" is a static constant -

[pentaho-platform-plugin-reporting/core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingAction.java](https://github.com/pentaho/pentaho-platform-plugin-reporting/blob/9b9b4a15605ce279d3321661f0dc061d3925832f/core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingAction.java#L102)

Line 102 in [9b9b4a1](https://github.com/pentaho/pentaho-platform-plugin-reporting/commit/9b9b4a15605ce279d3321661f0dc061d3925832f)

 public static final String USE_JCR = "useJcr";